pom.xml

```diff
@@ -135,5 +91,9 @@
             <scope>provided</scope>
         </dependency>
 
+        <dependency>
+            <groupId>org.projectlombok</groupId>
+            <artifactId>lombok</artifactId>
```

Comment:
This is _required_ to be an `optional` dependency it should _never_ come through transitively. We really should be validating that, I added it as a task here: https://github.com/BroadleafCommerce/jenkins-pipeline-library/issues/21